test/testenv/s3utils.go

```diff
@@ -64,27 +64,58 @@ func CheckPrefixExistsOnS3(prefix string) bool {
 	return false
 }
 
-// DownloadFromS3Bucket downloads license file from S3
-func DownloadFromS3Bucket() (string, error) {
-	dataBucket := testS3Bucket
+// DownloadLicenseFromS3Bucket downloads license file from S3
+func DownloadLicenseFromS3Bucket() (string, error) {
 	location := enterpriseLicenseLocation
 	item := "enterprise.lic"
-	file, err := os.Create(item)
+	filename, err := DownloadFileFromS3(item, location)
+	return filename, err
+}
+
+// S3Session Create session object for S3 bucket connection
+func S3Session() *session.Session {
+	sess, _ := session.NewSession(&aws.Config{Region: aws.String(s3Region)})
```

Comment:
If this errors out, is there no reason why we would need the error string?

Reply:
We should handle the error here and in callers as well.

Reply:
Modified